umbral/fragments.py

```diff
@@ -132,8 +130,7 @@ def to_bytes(self) -> bytes:
             + kfrag_sig2       \
             + sig              
 
-        if self.metadata is not None:
-            result = result + self.metadata
+        result += self.metadata or b''
```

Comment:
This is an odd line - the user is not allowed to use something falsey as metadata?